src/guide/jbs-jdk-bug-system.md

```diff
@@ -249,10 +249,27 @@ Once you are made, or you make yourself, the assignee of an issue you take on th
 
 Some additional fields should be filled out or updated as you get a better understanding of the issue:
 
-* For a regression, if you identify the fix that caused it, add a relates-to link to that issue (and add a [[regression_]{.jbs-label}(ID)](#regression_id) label) and set the [Introduced in Build]{.jbs-field} or [Introduced in Version]{.jbs-field} field.
 * The [Description]{.jbs-field} usually explains what went wrong and how the failure was found, then there's some investigation and eventually the root cause is found. At this point the [Summary]{.jbs-field} should be updated to correctly describe the bug. The [Description]{.jbs-field} however should remain a description of how the failure was found.
 * The [Affects Version/s]{.jbs-field} should be updated if you in your investigation finds that the issue is older than what is indicated by the current [Affects Version/s]{.jbs-field}.
 
+### Linking Issues
+
+An important aspect of any issue is making clear how it is connected/related to other issues. This can occur at any stage of the issue's lifecycle. For example, as information becomes available that might suggest a cause, or similar issue (relates to); or when a Backport or CSR request is created; or when closing as a duplicate of another issue.
+
+There are the following link types:
```

Comment:
I suggest marking up the issue type with bold `**` if not with `<dt>` and using a descriptive sentence maybe, for example:

> **‘duplicate of’** is normally set automatically when another issue is closed as a _‘duplicate’_, see \[Closing issues as duplicates\] for more information.
> 
> **‘backported by’** is set automatically when a fix is pushed to an older release or when a backport is created manually using the **More** -> **Create Backport** option.

The list would be easier to read if the description of all the link types followed the same pattern.

Reply:
should now be consistent